helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/ClusterModelProvider.java

```diff
@@ -97,27 +97,32 @@ public static ClusterModel generateClusterModel(ResourceControllerDataProvider d
    * Find the minimum set of replicas that need to be reassigned.
    * A replica needs to be reassigned if one of the following condition is true:
    * 1. Cluster topology (the cluster config / any instance config) has been updated.
-   * 2. The baseline assignment has been updated.
-   * 3. The resource config has been updated.
-   * 4. The resource idealstate has been updated. TODO remove this condition when all resource configurations are migrated to resource config.
-   * 5. If the current best possible assignment does not contain the partition's valid assignment.
+   * 2. The resource config has been updated.
+   * 3. If the current best possible assignment does not contain the partition's valid assignment.
    *
    * @param replicaMap             A map contains all the replicas grouped by resource name.
    * @param clusterChanges         A map contains all the important metadata updates that happened after the previous rebalance.
-   * @param activeInstances        All the instances that are alive and enabled.
+   * @param activeInstances        All the instances that are alive and enabled according to the delay rebalance configuration.
+   * @param liveInstances          All the instances that are alive.
```

Comment:
1. the znode is called liveInstance
2. we don't use aliveInstance in our code elsewhere. I believe all of the vars are named as "liveNode" or "liveInstance".

Reply:
What I was referring to was the description:

"All the instances that are alive" -> "All the instances that are live". "Live" would be proper since instances are not living organisms :)

Reply:
Sure, will modify.